### ISSUE TITLE
OSD keyboard accessibility issues

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#521]
Single image:
- cannot tab to the zoom controls
- image itself is a focusable element but doesn't have any actions associated

Multiple images:
- container (image plus sidebar) is focusable w no action
- can tab to the sidebar images, no action
- each sidebar image has two tabstops
- still can't tab to the zoom controls
